hyperdome/client/main.py

```diff
@@ -0,0 +1,132 @@
+# -*- coding: utf-8 -*-
+"""
+Hyperdome
+
+Copyright (C) 2019 Skyelar Craver <scravers@protonmail.com>
+                   and Steven Pitts <makusu2@gmail.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+import sys
+import argparse
+import signal
+from PyQt5 import QtCore, QtWidgets
+
+from ..common import strings
+from ..common.common import Common, platform_str
+from ..common.onion import Onion
+from ..server.hyperdome_server import HyperdomeServer
+from .hyperdome_client import HyperdomeClient
+
+
+class Application(QtWidgets.QApplication):
+    """
+    This is Qt's QApplication class. It has been overridden to support threads
+    and the quick keyboard shortcut.
+    """
+
+    def __init__(self, common):
+        if platform_str == "Linux" or platform_str == "BSD":
```

Comment:
I'm thinking this is old code, but
`if platform_str in ["Linux", "BSD"]:`

Reply:
old code that I want to replace with pathlib in the near future